src/javascript/JContent/JContent.accordion-items.jsx

```diff
@@ -1,28 +1,44 @@
 import React from 'react';
-import {File, FolderSpecial, Collections, Setting} from '@jahia/moonstone/dist/icons';
+import {Collections, File, FolderSpecial, Setting} from '@jahia/moonstone/dist/icons';
 import ContentTree from './ContentTree';
-import JContentConstants from './JContent.constants';
+import ContentRoute from './ContentRoute';
+import AdditionalAppsTree from './AdditionalAppsTree';
+import AdditionalAppsRoute from './AdditionalAppsRoute';
 
 export const jContentAccordionItems = registry => {
-    const renderDefaultContentTrees = registry.add('accordionItem', 'renderDefaultContentTrees', {
-        render: item => (
-            <ContentTree item={item}/>
-        ),
-        getPath: (site, pathElements, registryItem) => {
-            let path = '/sites/' + site + ('/' + pathElements.join('/'));
-            if (!path.startsWith('/sites/' + site + registryItem.config.rootPath)) {
-                return registryItem.defaultUrl(site);
-            }
+    const getPath = (site, pathElements, registryItem) => {
+        let path = '/sites/' + site + ('/' + pathElements.join('/'));
+        if (!path.startsWith('/sites/' + site + registryItem.config.rootPath)) {
+            return registryItem.defaultPath(site);
+        }
+
+        return path;
+    };
 
-            return path;
+    const getUrlPathPart = (site, path) => {
+        let sitePath = '/sites/' + site;
+
+        if (path.startsWith(sitePath + '/')) {
+            path = path.substring(('/sites/' + site).length);
+        } else {
+            path = '';
         }
+
+        return path;
+    };
+
+    const renderDefaultContentTrees = registry.add('accordionItem', 'renderDefaultContentTrees', {
+        component: ContentTree,
+        routeComponent: ContentRoute,
+        getPath,
+        getUrlPathPart
     });
 
-    registry.add('accordionItem', JContentConstants.mode.PAGES, renderDefaultContentTrees, {
+    registry.add('accordionItem', 'pages', renderDefaultContentTrees, {
```

Comment:
Why don't you keep the constant here?

Reply:
Good question :-) The tab id should be configurable, and we could add as many tabs as we wants - so basically the app name should not be used anywhere else, hence not be a constant (since it's dynamic ...). Unfortunately the constant is still used in different places, and some code is still harcoding usage of some accordions .. but we should get rid of it. I arleady removed  the "apps" constants.

Reply:
Ok that's what I thought when seeing the rest of the code.

Good to me then!